tests/conftest.py

```diff
@@ -25,6 +25,13 @@
 from tests import today
 
 
+def pytest_configure(config):
+    env_values = {"SECRETS_FOR_DYNACONF": "mobilizon_reshare/.secrets.toml"}
+    for key in env_values:
+        if not os.environ.get(key):
```

Comment:
i test devono essere deterministici quindi questa condizione secondo me crea più problemi che altro. Se un utente specifica per sbaglio dell envvar che vanno in conflitto coi test, è un motivo ulteriore per sovrascriverle.